app/feed/process_alignment_feed.py

```diff
@@ -1,38 +1,100 @@
+    The effects are not yet assigned appropriately.
 
     Parameters
     ==============
     conversation_uuid (UUID)
     quiz_uuid (UUID) - user b quiz uuid to compare scores with user a scores
     alignment_feed_uuid (UUID) - uuid created when post alignment endpoint is used
     """
-    # TODO: Add logic to create aligned feed. Currently working with hard-coded dummy values.
 
     try:
         alignment_feed = AlignmentFeed()
         alignment_feed.alignment_feed_uuid = alignment_feed_uuid
-        alignment_feed.aligned_effect_1_iri = "R9JAWzfiZ9haeNhHiCpTWkr"
-        alignment_feed.aligned_effect_2_iri = "R8JoXNnKTYqERwU7fblKTWB"
-        alignment_feed.aligned_effect_3_iri = "RB7k7p2iQQgKdQrkRP2MZWM"
-        alignment_feed.aligned_solution_1_iri = "RBeBCvukdLNSe5AtnlJpQ1k"
-        alignment_feed.aligned_solution_2_iri = "R9SuseoJG7H6QeUEvZwLciQ"
-        alignment_feed.aligned_solution_3_iri = "R9R6552i4fn3XHKpoV8QTOx"
-        alignment_feed.aligned_solution_4_iri = "RDSZw453Ge76hYTvYEsaAwU"
-        alignment_feed.aligned_solution_5_iri = "RDanTqMAQyQ4nGzlrt0j7Bm"
-        alignment_feed.aligned_solution_6_iri = "RItKzuJSSFw9hXydUSVEJX"
-        alignment_feed.aligned_solution_7_iri = "RBCQdAOKui38ytAIKZlpPN6"
+        assign_alignment_iris(alignment_feed, "effect", find_alignment_effect_iris())
+        assign_alignment_iris(
+            alignment_feed, "solution", find_alignment_solution_iris()
+        )
         db.session.add(alignment_feed)
         db.session.commit
     except:
         raise DatabaseError(
             message="An error occurred while adding the alignment feed to the database."
         )
+
+
+def assign_alignment_iris(alignment_feed, field_type, iris):
+    """Set the solution iri fields in the alignment feed."""
+    for (index, iri) in enumerate(iris, start=1):
+        setattr(alignment_feed, "aligned_{}_{}_iri".format(field_type, index), iri)
+
+
+def find_alignment_effect_iris():
+    # TODO: Add logic. Currently working with hard-coded dummy values.
+    return [
+        "R9JAWzfiZ9haeNhHiCpTWkr",
+        "R8JoXNnKTYqERwU7fblKTWB",
+        "RB7k7p2iQQgKdQrkRP2MZWM",
+    ]
+
+
+def find_alignment_solution_iris():
+    """Choose and order solutions for the alignment.
+
+    Using the (mitigation) solutions from the ontology, put the conversation solution first,
+    followed by a random ordering of POPULAR_SOLUTION_COUNT solutions from POPULAR_SOLUTION_NAMES
+    and UNPOPULAR_SOLUTION_COUNT other solutions. This function takes no arguments, since solutions
+    are (currently) independent of users' personal values etc.
+
+    Returns
+    ==========
+    List of strings: an ordered list of solution iris for an alignment feed
+    """
+    solution_map = {"talk": None, "popular": set(), "unpopular": set()}
+    solution_nodes = get_solution_nodes()
+    for node in solution_nodes:
+        name = node["label"]
+        iri = node["iri"][len("webprotege.stanford.edu.") :]

```

Comment:
There’s a helper function already somewhere in the codebase to do this line. You should just use it here since it is possible that the iri base could change in the future. It will minimize the places need to change webprotege.stanford.edu